### PR TITLE
Dmytro.ovcharov

### PR DESCRIPTION
**Pull request by Jammer77**
53532 bug fixed Empty "Label" for edit fields in "Solution Options" window

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects.OptionPanels/CombineBuildOptions.cs [hunk 1]
```diff
@@ -54,6 +54,19 @@ public CombineBuildOptionsWidget (Solution solution)
 			Build ();
 			this.solution = solution;
 			folderEntry.Path = solution.OutputDirectory;
+<<<<<<< HEAD
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects.OptionPanels/CombineBuildOptions.cs]: Please remove the merge conflict markers.

**Comment by Jammer77** [comment]
fix 53532

File: main/src/core/MonoDevelop.Ide/gtk-gui/MonoDevelop.Ide.Projects.OptionPanels.CombineBuildOptionsWidget.cs [hunk 2]
```diff
@@ -2,65 +2,70 @@
 // This file has been generated by the GUI designer. Do not modify.
 namespace MonoDevelop.Ide.Projects.OptionPanels
 {
-	internal partial class CombineBuildOptionsWidget
+	public partial class CombineBuildOptionsWidget
```
Comment (mrward) [inline main/src/core/MonoDevelop.Ide/gtk-gui/MonoDevelop.Ide.Projects.OptionPanels.CombineBuildOptionsWidget.cs]: This probably should not be public.

Comment (Jammer77) [inline main/src/core/MonoDevelop.Ide/gtk-gui/MonoDevelop.Ide.Projects.OptionPanels.CombineBuildOptionsWidget.cs]: I fixed it!

**Pull request merged**